CRM/Core/Form.php

```diff
@@ -1580,6 +1580,9 @@ public function addDefaultButtons($title, $nextType = 'next', $backType = 'back'
   }
 
   /**
+   * @deprecated
```

Comment:
Not gonna block this over it but I have been ensuring deprecated tags look like

@deprecated since 5.45 will be removed around 5.90

Generally I put some small amount of thought into the will be removed - ie if I think it is not really used / broken I might go for 6 months but if there is a bit of usage / it's not causing lots of grep/ pain then a longer date